src/workerd/api/worker-rpc.c++

```diff
@@ -966,12 +966,29 @@ MakeCallPipeline::Result serializeJsValueWithPipeline(jsg::Lock& js,
 // session.
 class JsRpcTargetBase: public rpc::JsRpcTarget::Server {
  public:
-  JsRpcTargetBase(IoContext& ctx)
+  struct MayOutliveIncomingRequest {};
```

Comment:
This feels really weird, but I guess there's not really a better way to do it because it's for constructors.